pkg/apis/elasticsearch/v1alpha1/types.go

```diff
@@ -56,6 +56,31 @@ type ElasticsearchNode struct {
 	Storage      ElasticsearchNodeStorageSource `json:"storage"`
 }
 
+type ElasticsearchStorageSpec struct {
```

Comment:
Where is the size spec for the volume?

Reply:
Will add to Host and StorageClass structs

Reply:
Why not add them as a subfield to this spec?  Is it for the case of using a pre-existing PVC where there is not a need to define the storage?